api/kyverno/v1/common_types.go

```diff
@@ -86,6 +86,11 @@ type ContextEntry struct {
 	// ConfigMap is the ConfigMap reference.
 	ConfigMap *ConfigMapReference `json:"configMap,omitempty" yaml:"configMap,omitempty"`
 
+	// Default is an optional arbitrary JSON object that the context may take if the apiCall
+	// returns error
+	// +optional
+	Default *apiextv1.JSON `json:"default,omitempty" yaml:"default,omitempty"`
```

Comment:
Why do you add the `Default` directly in `ContextEntry`? Is this the default to all or just `apiCall`?

Reply:
Yes, `Default` is just for `apiCall`. I am sorry I got confused with the issue #9723.  As the solution yaml demands for `Default` in `ContextEntry` . Should I add default inside apiCall ?